dcm4che-imageio/src/main/java/org/dcm4che3/imageio/stream/SegmentedImageInputStream.java

```diff
@@ -257,4 +257,22 @@ public long getLastSegmentEnd() {
             return bulk.getSegmentEnd();
         }
     }
+
+    /**
+     * Determine the length of the total segmented stream.
+     */
```

Comment:
It's *VERY* useful to know the length of a stream, so I have added length() into all of the IIS implementations.